app/views/users/show.html.erb

```diff
@@ -1 +1,35 @@
-<h1>Here is a list of a specific user</h1>
\ No newline at end of file
+<%= render "shared/header" %>
+    <div class="show-user-container">
+ <div class="show-header">
+    <img src="<%= @user.photo %>" alt="user photo">
+    <div>
+       <h2>Name: <%= @user.name %></h2>
+      <h4>Number of posts: <%= @user.post_counter %></h4>
+    </div>
+  </div>
+<% end %>
+</div>
+ <div class="bio">
+    <h2>Bio</h2>
+    <p><%= @user.bio %></p>
+ </div>
+
+ <% @posts.each do |post| %>
+    <div>
+      <div class="show-posts">
+         <h2><%= post.title %></h2>
+         <p><%= post.text %></p>
+      <div>
+      <div>
+        <span>Comments: <%= post.comments_counter %></span>
+        <span>Likes: <%= post.likes_counter %></span>
+      </div>
+    </div>     
+ <% end %>
+</div>
+
+<button type="button">
+  <%= link_to "See all posts", user_posts_path(@user) %>
+</button>
```

Comment:
- [ ] As per project requirements, please make sure to use the recent_posts method to render the most recent 3 posts for each user, and this would allow us to logically press 'See all posts' button to see the rest of the posts of that user. 👍